appservice/src/utils/azureUtils.ts

```diff
@@ -0,0 +1,15 @@
+
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+// tslint:disable-next-line: export-name
+export function areLocationNamesEqual(name1: string | undefined, name2: string | undefined): boolean {
+    return normalizeLocationName(name1) === normalizeLocationName(name2);
+}
+
+function normalizeLocationName(name: string | undefined): string {
```

Comment:
Can we somehow just use LocationListStep's `generalizeLocationName` instead of having a util file for this?

Reply:
We could, but this code felt more like `delay` or `nonNull` - aka little utils that don't quite make sense to be exposed in the UI package, so we just copy it to every repo we need it in